### PR TITLE
add `verbatim` as alias of `raw` for parity w/ Twig

### PR DESCRIPTION
**Pull request by jbmoelker**
## Summary

Proposed change:

* Add `verbatim` as an alias of `raw` for compatibility with Twig (see [Twig `verbatim` tag](http://twig.sensiolabs.org/doc/tags/verbatim.html)).
* Extends `parseRaw` with `tagName` parameter to support `verbatim`.
* Reuses `raw` tag tests.
* Explains compatibility in docs.

## Checklist

I've completed the checklist below to ensure I didn't forget anything. This makes reviewing this PR as easy as possible for the maintainers. And it gets this change released as soon as possible.

* [x] Proposed change helps towards [*purpose of this project*](https://github.com/mozilla/nunjucks/blob/master/CONTRIBUTING.md#purpose).
* [x] [*Documentation*](https://github.com/mozilla/nunjucks/tree/master/docs/) is added / updated to describe proposed change.
* [x] [*Tests*](https://github.com/mozilla/nunjucks/tree/master/tests) are added / updated to cover proposed change.
* [x] [*Changelog*](https://github.com/mozilla/nunjucks/blob/master/CHANGELOG.md) has an entry for proposed change (if user-facing fix or feature).

(Tick of items by replacing `[ ]` by `[x]`)

### CODE REVIEW

File: src/parser.js [hunk 1]
```diff
@@ -575,9 +576,12 @@ var Parser = Object.extend({
         return node;
     },
 
-    parseRaw: function() {
+    parseRaw: function(tagName) {
+        tagName = tagName || 'raw';
+        var endTagName = 'end' + tagName;
         // Look for upcoming raw blocks (ignore all other kinds of blocks)
-        var rawBlockRegex = /([\s\S]*?){%\s*(raw|endraw)\s*(?=%})%}/;
+        //var rawBlockRegex = /([\s\S]*?){%\s*(raw|endraw)\s*(?=%})%}/;
```
Comment (vecmezoni) [inline src/parser.js]: remove this and this PR is ready to be merged

**Comment by ricordisamoa** [comment]
> There should be one-- and preferably only one --obvious way to do it.

The Zen of Python

**Comment by jbmoelker** [comment]
@vecmezoni I removed forgotten comment and resolved merge conflicts.

**Comment by vecmezoni** [comment]
👍

**Pull request merged**